content/transactions/checkcash.md

```diff
@@ -0,0 +1,39 @@
+## CheckCash
+[[Source]<br>](https://github.com/ripple/rippled/blob/master/src/ripple/app/tx/impl/CashCheck.cpp "Source")
```

Comment:
Double checking that the link is as intended. I see the page on `develop`.

https://github.com/ripple/rippled/blob/master/src/ripple/app/tx/impl/CashCheck.cpp

Reply:
It'll be in master after the release, but pointed it to develop for now so the link works before then